### ISSUE TITLE
[FineTuning]

### ISSUE THREAD

**Issue by sunjiwon** [CederGroupHub/chgnet#165]
Hello everyone,

I have a question regarding the fine-tuning process of CHGNet.

If I retrain CHGNet using energy, force, and stress data, will the model still utilize the magmom (magnetic moments) after the retraining process?

I would appreciate your insights on this matter.

Thank you and have a nice day!

**Comment by BowenD-UCB** [comment]
The model will still give a MAGMOM prediction,  but please be cautious with the MAGMOMs predicted if you perform fine-tuning without MAGMOM labels.
I would recommend performing some benchmarks with DFT, since neural networks are known to 'forget' after training on new labels.

**Comment by sunjiwon** [comment]
Hello everyone,

Thank you for your previous help.

I have one more question. As mentioned before, neural networks are known to "forget" after training on new labels. Does this mean that CHGNet will function like transfer learning and lose all its memory of the MP data?

I appreciate your insights on this matter.

Thank you again!
